tools/common/Frameworks.cs

```diff
@@ -188,6 +188,7 @@ public Framework Find (string framework)
 					{ "PencilKit", "PencilKit", 10,15 },
 					{ "Speech", "Speech", 10,15 },
 					{ "LinkPresentation", "LinkPresentation", 10,15 },
+					{ "CoreHaptics", "CoreHaptics", 10,15 },
```

Comment:
You have a merge conflict here; when fixing it make sure to keep this list alphabetically ordered.